controllers/RatingsController.js

```diff
@@ -0,0 +1,76 @@
+import model from '../models';
+import StatusResponse from '../helpers/StatusResponse';
+
+const { ratings } = model;
+
+/**
+ * @description - This class is all about users ratings of articles
+ * @param {object} req
+ * @param {object} res
+ * @returns {class} Users
+ */
+class RatingsController {
+  /**
+   * @description - This method takes care of creating ratings on an article by a user
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} Users Ratings on an article
+   */
+  static async create(req, res) {
+    try {
+      const usersRatings = await ratings.create({
+        userId: req.userId,
+        identifier: req.params.identifier,
+        stars: req.body.stars
+      });
+      StatusResponse.created(res, {
+        message: 'Users ratings on this article recorded succesfully',
+        ratings: usersRatings
+      });
+    } catch (error) {
+      StatusResponse.internalServerError(res, {
+        message: 'users ratings not recorded succesfully, please try again',
+        error: {
+          body: [`Internal server error => ${error}`]
+        }
+      });
+    }
+  }
+
+  /**
+   * @description - This method takes care of retrieving ratings on an article by all users
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} Ratings on an article
+   */
+  static async retrieve(req, res) {
+    try {
+      console.log(req.params.identifier)
+      const returnedRatings = await ratings.findAll({
```

Comment:
Do a findAndCountAll so we just use the count result instead of doing an `[].length`.
Also, if what we need is just the number of stars and number of people who created stars, we should fetch just what we need for the maths.

Reply:
Feedback Noted @Chidi3b3r3